### PR TITLE
docs: update Home Assistant integration documentation with configuration URL and model name hints

### PR DESCRIPTION
**Pull request by JakobLichterfeld**
fixes #4351

### CODE REVIEW

**Review by swiffer — APPROVED** [review]
looks good to me

File: website/docs/integrations/home_assistant.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ Don't forget to replace `<teslamate url>` and `<your tesla model>` with correct
         payload_not_available: "false"
     device: &teslamate_device_info
       identifiers: [teslamate_car_1]
-      configuration_url: https://teslamate.zxxz.io/
+      configuration_url: https://teslamate.zxxz.io/ # update this with your teslamate URL
```
Comment (swiffer) [inline website/docs/integrations/home_assistant.md:69]: @JakobLichterfeld - alternative suggestion

<teslamate url> # update this with your teslamate URL, e.g. https://teslamate.zxxz.io/

that way users can see search for the placeholders (same for the other placeholders).

but both changes fine for me

Comment (JakobLichterfeld) [reply website/docs/integrations/home_assistant.md]: You are absolutely right and I have been lazy. 🙂

File: website/docs/integrations/home_assistant.md [hunk 2]
```diff
@@ -66,10 +72,10 @@ Don't forget to replace `<teslamate url>` and `<your tesla model>` with correct
         payload_not_available: "false"
     device: &teslamate_device_info
       identifiers: [teslamate_car_1]
-      configuration_url: https://teslamate.zxxz.io/ # update this with your teslamate URL
+      configuration_url: <teslamate url> # update this with your teslamate URL, e.g. https://teslamate.example.com/
       manufacturer: Tesla
-      model: Model 3 # update this with your car model
-      name: Tesla Model 3 # update this with your car name
+      model: Model 3 # <your tesla model>, update this with your car model
+      name: Tesla Model 3 # <your tesla name> update this with your car name
```
Comment (swiffer) [inline website/docs/integrations/home_assistant.md:78]: <your tesla ...> should be in the yml, Model 3 and Tesla Model 3 in the comment as example ?

Comment (JakobLichterfeld) [reply website/docs/integrations/home_assistant.md:78]: Indeed, same as above, laziness.

**Pull request merged**